dotnet-interactive/CommandLine/CommandLineParser.cs

```diff
@@ -153,6 +153,11 @@ public static class CommandLineParser
                 "--log-path",
                 "Enable file logging to the specified directory");
 
+            var pathOption = new Option<DirectoryInfo>(
+                "--path",
+                "Installs the kernelspecs to the specified directory")
+                .ExistingOnly();
```

Comment:
Is `ExistingOnly` really needed?  Would it be more consistent to behave like `--log-path`? I guess the log directory will be created if it doesn't exist.

Reply:
Ideally this directory exists and was created by some flavor of Jupyter install. If the directory doesn't exist, and we create it on the fly, it seems like it increases the likelihood that Jupyter won't find the kernelspecs.

Reply:
If the user specifies the path, then we probably should trust the user knows what he is doing :) Otherwise, the user should depends on the default path.

Reply:
Creating a directory is pointless if Jupyter doesn't know about it. Maybe I want to run the `dotnet interactive jupyter install` before I've installed Jupyter but it seems like an edge case, won't block anyone, and adds complexity, so I'm happy to keep it simpler until we have more feedback.